cookie-banner-rules-list.json

```diff
@@ -665,13 +659,11 @@
       "domains": ["yandex.com", "yandex.ru", "ya.ru", "kinopoisk.ru"]
     },
     {
-      "click": {},
-      "cookies": {
-        "optOut": [
-          { "name": "euconsent-v2", "value": "1" },
-          { "name": "euconsent-v2-noniab", "value": "AAYA" }
-        ]
+      "click": {
+        "optIn": ".components-button.white-space-normal.is-primary",
+        "presence": ".components-modal__screen-overlay"
```

Comment:
This is too generic, can you find a click rule that is more specific to the cookie banner?

Reply:
Outside of parsing the text within the button, there is nothing more specific to banner

Reply:
In that case we should remove the rule. We shouldn't risk clicking the wrong thing.
The class `.cmp__dialog` seems useful, but the buttons only differ in text and role. Targeting based on role is too weak IMO.